41.md

```diff
@@ -0,0 +1,289 @@
+NIP-41
+======
+
+Identity management
+--------------------
+
+`draft` `optional`
+
+
+This NIP introduces a reliable and verifiable solution for users to manage their identities, along with standard procedures for dealing with catastrophic cases. It also outlines the differences between simple and secure identities.
+
+# Basic concepts 
+* **Secured identities**
+	* These identities are composed of a **master keypair** and an **active subkey**.
+	* The master key pair is used to derive, secure and back up subkeys, and acts as a trusted source of truth for announcements related to subkeys.
+	* To derive a subkey it is recommended, but not mandatory, to use bip-32 as described in [NIP-06](https://github.com/nostr-protocol/nips/blob/master/06.md) with the derivation path `m/44'/1237'/<account>'/0/0`. This gives us HD addresses and a `xpub` that can be used as proof in a future catastrophe case. Other derivation paths can be used if the user knows what its doing. No derivation scheme needs to be used, users can just generate random keypairs and use them, this gives users less control over subkeys and no proof of derivation.
```

Comment:
Ouch, thanks for your observation, it's absolutely true. The solution I think would then be to use subkeys derived from non-hardened derivation paths, for example `m/44'/1237'/0'/0`, and if any of the subkeys in use are compromised, derive a new `account`, for example `m/44'/1237'/1'`, and derive non-hardened subkeys from that, for example `m/44'/1237'/1'/0`, so we could maintain the security of the master key pair. That is, if we want to keep the proof of the derivation tree. Otherwise, this proof would be meaningless, as revealing the `xpub` would not recreate the subkeys obtained from hardened levels of the path.

Reply:
Avoided derivation tree proof for now to reduce complexity of the scheme, as it was optional